bridges/snowbridge/primitives/router/src/outbound/mod.rs

```diff
@@ -20,6 +20,8 @@ use sp_std::{iter::Peekable, marker::PhantomData, prelude::*};
 use xcm::prelude::*;
 use xcm_executor::traits::{ConvertLocation, ExportXcm};
 
+pub const ASSET_HUB_PARA_ID:u32 = 1000;
```

Comment:
It is an anti-pattern to hardcode para-id constants in these types of primitives that could be in theory reused in other scenarios.

In practice this is fine, but the used pattern is to make this a generic parameter so it can be customized by the runtime.

Reply:
Thanks. Yeah, just a workaround before getting some feedback. 

Fixed in https://github.com/paritytech/polkadot-sdk/pull/6838/commits/277d51b77a0a5f04dff29ae47fb9fbb51c9a944c